spec/unit/provider/directory_spec.rb

```diff
@@ -197,6 +197,37 @@
         expect { directory.run_action(:create) }.to raise_error(Chef::Exceptions::EnclosingDirectoryDoesNotExist)
       end
     end
+
+    describe "on OS X", mac_osx_only: true do
```

Comment:
Yes, please. We use these tags for tests that can only pass on the given platform, i.e., the non-mocked tests in `spec/functional/`